compliance_checker/acdd.py

```diff
@@ -516,19 +521,21 @@ def verify_convention_version(self, ds):
         """
         Verify that the version in the Conventions field is correct
         """
-        for convention in ds.Conventions.replace(' ', '').split(','):
-            if convention == 'ACDD-' + self._cc_spec_version:
-                return ratable_result((2, 2),
-                                      'Conventions',
-                                      [])
-        # Conventions attribute is present, but does not include
-        # proper ACDD version
-        messages = [
-            "Global Attribute 'Conventions' does not contain 'ACDD-{}'".format(self._cc_spec_version)
-        ]
-        return ratable_result((1, 2),
-                              'Conventions',
-                              messages)
+        try:
+            for convention in ds.getncattr("Conventions").replace(' ', '').split(','):
+                if convention == 'ACDD-' + self._cc_spec_version:
+                    return ratable_result((2, 2), None, []) # name=None so grouped with Globals
+
+                else:
```

Comment:
This `else... return` statement should actually be outside of the `for` loop, as datasets can have multiple conventions.